src/frag.js

```diff
@@ -215,6 +223,14 @@ function insertBefore(
 }
 
 function appendChild(node) {
+	if (
```

Comment:
Can you add a comment above these checks explaining what it's doing and why

Reply:
Seeking your assistance here, I find it difficult to provide an in-depth explanation of what exactly is going on.

With `keep-alive`, there are two consecutive inserts, the second one is "falsy" and should be ignored. The only property they don't have in common is `parentElement`, so checking if it is defined is the first part of the expression. The second part is tbh needed to prevent everything else from breaking and pass tests

Reply:
Documented

Reply:
Thanks